WordPress/WordPressTest/ReaderDetailCoordinatorTests.swift

```diff
@@ -355,6 +355,10 @@ private class ReaderDetailViewMock: UIViewController, ReaderDetailView {
         didCallScrollToWith = to
     }
 
+    func updateHeader() {
+
+    }
+
```

Comment:
Empty implementation? Maybe add a comment so it doesn't look incomplete?
```suggestion
    // Empty implementation to adhere to the protocol
    func updateHeader() { }
```

Reply:
Done in 7d7047a7a2053583a31bf71411ac1b1a7e90e37a